app/controllers/form_controller.rb

```diff
@@ -71,7 +71,7 @@ def restore_error_field_values
     end
     if session["fields"]
       session["fields"].each do |field, value|
-        if form.get_question(field, @log)&.type != "date" && @log.respond_to?(field)
+        if form.get_question(field, @log)&.type != "date" && @log.attributes.key?(field)
```

Comment:
Will this fix the issue if there is an attribute in both sales and lettings logs e.g. age1? Couldn't there be an error added to age1 in a lettings log that would then appear on a sales log if the sales log age1 question was open in a different tab? Chance of this happening is small to be fair